### PR TITLE
Human only ERT and Deathsquad

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Now tied to the human authority config option

### CODE REVIEW

**Comment by PKPenguin321** [comment]
yee

**Comment by Shadowlight213** [comment]
That's a lot of commits.

**Comment by KorPhaeron** [comment]
@Shadowlight213 

I'm lazy and use git for desktop so it makes one of those everytime I update I should probably use command line

**Comment by Steelpoint** [comment]
Merge post haste :+1: 

Should include Nuke Ops as well, maybe as a separate config if need be to avoid a 50 comment war.

**Comment by HankMoodyExperience** [comment]
:+1:

**Comment by RemieRichards** [comment]
@KorPhaeron use tortoisegit, we have a buttload of guides specific to it on the wiki.

**Comment by Zilenan91** [comment]
obligatory :-1:

**Comment by Bawhoppen** [comment]
I still don't know why people complained about no lizards of flops teams

**Comment by Hatfish** [comment]
Does this assign a random human body to lizard players or does it just prevent someone with a lizard character selected from playing one of these roles?

**Comment by KorPhaeron** [comment]
@Hatfish 

Makes the lizard player into a human

**Comment by Hatfish** [comment]
In that case, :+1:

**Comment by SweaterKittens** [comment]
@Bawhoppen and @Steelpoint 

It was decided that it wasn't unreasonable to believe that the syndicate would employ nearly anyone they could who was willing to wage war on Nanotrasen, lizards included - while it would be unrealistic to have "lower-class" citizens in ERT and Deathsquad (typically high ranking teams) working for Nano.

**Comment by Bawhoppen** [comment]
@SweaterKittens 'it was decided'
I wasn't the only person to argue that the syndies are racist too.

**Comment by KorPhaeron** [comment]
I still dont want them on nuke teams either since they're basically syndie deathsquads and not at all rag tag but I'll keep that for another PR

**Comment by SweaterKittens** [comment]
@Bawhoppen 
That's fair, I was trying to put words in your mouth, just that the overall vibe I got from that comment thread was that people were mostly accepting of them as nuke operatives.

**Comment by Steelpoint** [comment]
The whole non-human Op issue seemed hostly contested, not one sided.

In fact that whole PR become a cluster fuck to follow.

**Comment by Dorsisdwarf** [comment]
The clusterfuckness is why Kor made this one seperately

File: html/changelogs/KOR-lizards.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Cheridan) [inline html/changelogs/KOR-lizards.yml]: thanks for the PR pomf

Comment (Shadowlight213) [inline html/changelogs/KOR-lizards.yml]: Kor is pomf confirmed.

**Comment by KorPhaeron** [comment]
Changelog fixed the hate of lizards is now properly credited

**Pull request merged**